conda_forge_tick/migrators/version.py

```diff
@@ -445,19 +445,14 @@ def migrate(
 
         try:
             with open(os.path.join(recipe_dir, "meta.yaml"), "r") as fp:
-                cmeta = CondaMetaYAML(fp.read())
+                old_meta_yaml = fp.read()
+                cmeta = CondaMetaYAML(old_meta_yaml)
         except Exception as e:
             attrs["new_version_errors"][version] = (
                 "We found a problem parsing the recipe: \n\n" + str(e)
             )
             return {}
 
-        # cache round-tripped yaml for testing later
-        s = io.StringIO()
-        cmeta.dump(s)
-        s.seek(0)
-        old_meta_yaml = s.read()
```

Comment:
The cached yaml here needs to be round tripped due to how rumael.yaml does parsing and makes changes.